purchase_order_secondary_unit/models/purchase_order.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2018 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import api, fields, models
+
+
+class PurchaseOrderLine(models.Model):
+    _inherit = ["purchase.order.line", "product.secondary.unit.mixin"]
+    _name = "purchase.order.line"
+    _secondary_unit_fields = {
+        "qty_field": "product_qty",
+        "uom_field": "product_uom",
+    }
+    _product_uom_field = "uom_po_id"
+
+    product_qty = fields.Float(precompute=True)
```

Comment:
I already have this issue. To avoid the warning, we have to duplicate some attributes (like already discussed here: https://github.com/OCA/purchase-workflow/pull/2068/files#r1415171629 ).
I agree with the comment: we shouldn't have to duplicate attribute. So we have to do a choice between duplicates and the warning.

Reply:
Restored duplicated attributes in order to avoid warnings

Reply:
Thanks